src/vma/proto/neighbour.h

```diff
@@ -350,6 +350,10 @@ class neigh_entry : public cache_entry_subject<neigh_key, neigh_val *>, public e
 private:
 	bool 			m_is_first_send_arp;
 
+	const uint32_t	m_n_neigh_wait_till_send_arp_msec;
```

Comment:
formatting: please remove extra line and align member with space/tab to other memebrs